javatests/arcs/core/data/expression/ExpressionTest.kt

```diff
@@ -293,6 +303,32 @@ class ExpressionTest {
         )
     }
 
+    @Test
+    @Suppress("UNCHECKED_CAST")
+    fun evaluate_paxel_orderby() {
```

Comment:
Defining scope data in this test case instead of adding it to the scope at the top of the file would make this test more self contained and more readable Would you care to move the "names" and "parents" to this test case? You can see I've been doing this already for some tests e.g. `evaluate_fieldOps_nullSafe`.

Reply:
Done